### DISCUSSION TITLE
How to setup source maps correctly ?

### DISCUSSION THREAD

**Discussion by alexisvisco** [vercel/next.js#18623] [Help]
Hello :hand: 

I have a problem with source maps in [my next js application](https://github.com/asvf-montagne/client).

So I got an error: 
![image](https://user-images.githubusercontent.com/17145294/97807042-dfab9680-1c5e-11eb-9a3e-27f2641e4de5.png)

But when I am clicking on a source file it's not the original source file, it's the built source file, the ugly one.
![image](https://user-images.githubusercontent.com/17145294/97807085-23060500-1c5f-11eb-9761-d1838640e650.png)

So is it my configuration which is wrong? Because I set up the nextjs app with the generator ...

**Comment by alexisvisco** [comment]
Up, nobody knows the issue?

**Reply by ray1412** [reply]
AFAIK, if you use NextJS version < 10, you can use https://www.npmjs.com/package/@zeit/next-source-maps. Otherwise, there is `productionBrowserSourceMaps` key to be applied inside `next.config.js`
